app/jobs/articles/analytics_updater_job.rb

```diff
@@ -0,0 +1,11 @@
+module Articles
+  class AnalyticsUpdaterJob < ApplicationJob
+    queue_as :article_analytics_updater_job
+
+    def perform(user_id, analytics_updater = Articles::AnalyticsUpdater)
+      return unless User.find_by(id: user_id)
+
+      analytics_updater.call(user_id)
```

Comment:
you can pass the user object to the service here, it's just the serialized job that shouldn't have a user

Reply:
I have updated in that way.